README.md

```diff
@@ -1,39 +1,33 @@
 # Athena
-[![Build Status](https://travis-ci.org/Blacksmoke16/athena.svg?branch=master)](https://travis-ci.org/Blacksmoke16/athena)
+[![Build Status](https://img.shields.io/github/workflow/status/blacksmoke16/athena/CI)
 [![Latest release](https://img.shields.io/github/release/Blacksmoke16/athena.svg?style=flat-square)](https://github.com/Blacksmoke16/athena/releases)
 
-Modular annotation based web framework with built in param conversion inspired by [Symfony Routing](https://symfony.com/doc/current/routing.html).
-
-## Goals
-
-- Extensible and customizable to fit all use cases.
-- Be easy to adopt and start using effectively.
-- Remove unneeded boilerplate code to convert route/body params to correct type.
-- Easy serialization via [CrSerializer](https://github.com/Blacksmoke16/CrSerializer).
-- Decrease bloat.  Only include what you need.
-
-## Documentation
-
-[Documentation](./docs)
+A web framework comprised of reusable, independent components.
 
 ## Installation
 
-Add this to your application's `shard.yml`:
+1. Add the dependency to your `shard.yml`:
 
 ```yaml
 dependencies:
   athena:
-    github: Blacksmoke16/athena
+    github: blacksmoke16/athena
 ```
 
+2. Run `shards install`
+
+## Documentation
+
+Everything is documented in the [API Docs](https://athena-framework.github.io/config/Athena/Routing.html).
```

Comment:
Is this correct URL? 404 Not Found.

Reply:
It will be when it's migrated to the org, however that'll happen in another PR so i'll update it until then.